docs/team/benjytan45678.md

```diff
@@ -2,34 +2,47 @@
 layout: page
 title: benjytan45678's Project Portfolio Page
 ---
+
 ### Project: TrackAScholar
 
+TrackAScholar is a desktop application used for managing scholarship applications.
+The user interacts with it using a CLI, and it has a GUI created with JavaFX.
+It is written in Java, and has about 14 kLoC.
 
 Given below are my contributions to the project.
 
-* **New Feature**: Added `FilterCommand` and `ApplicationStatusPredicate` class under `applicant` package with Jie Hui
+* **New Feature**: Added `FilterCommand` and `ApplicationStatusPredicate` class under `applicant` package with Jie Hui: [#57](https://github.com/AY2223S1-CS2103T-W10-3/tp/pull/57) and [#67](https://github.com/AY2223S1-CS2103T-W10-3/tp/pull/67).
+    * Justification: Allows administrative staff to manage scholarship applicants more easily by organising them based on their specific application status.
 
-* **New Feature**: Added `PinCommand` and `UnPinCommand` class under `applicant` package with its respective parsers
+* **New Feature**: Added `PinCommand` and `UnPinCommand` class under `applicant` package with its respective parsers: [#105](https://github.com/AY2223S1-CS2103T-W10-3/tp/pull/105), [#115](https://github.com/AY2223S1-CS2103T-W10-3/tp/pull/115) and [#116](https://github.com/AY2223S1-CS2103T-W10-3/tp/pull/116)
```

Comment:
Maybe abstract out the actual class and write it like:
Added the ability to filter applicants by their scholarship application status...

Also you can leave jie hui's name in the credit if you want to mention that you developed it with him. Something like:
Credit: Feature developed in tandem with Jie Hui.